src/engine/SCons/Tool/msvsTests.py

```diff
@@ -755,6 +764,9 @@ def normalizeParam(param):
             class _DummyEnv(DummyEnv):
                 def subst(self, string, *args, **kwargs):
                     return string
+
+                def Dir(env, name):
+                    return self.fs.Dir(name)
```

Comment:
Does this actually create the dir when you run the test?

Reply:
Ok I confirmed that this doesn't create the directory, which is what I want.

That said, I think your pull request is using self.fs as though it is a member of _DummyEnv when it is only being created as a member of the msvsTestCase object. I will update my pull request with most of your change but I will need to add a fs object to DummyEnv.